### ISSUE TITLE
Add animations support

### ISSUE THREAD

**Issue by NathV** [amakaseev/sprite-sheet-packer#48]
This program has a cocos2d.js file for creating a .plist spritesheet file, but what about animations ? 

We can add new defaultFormats or an "Auto-detect Animation" option (sprites ending with a number will be grouped in animations in the data file) as in TexturePacker.

**Comment by amakaseev** [comment]
Now there is only a preview of animation :(
Possibly in this window to make animation control (speed of playing and others)
And to use these data for publish in Cocos2D and Json?

I don't know as TexturePacker keeps animation, it will be necessary to look from where it takes settings

**Comment by NathV** [comment]
We can do it if we want the user to have control over the options.
We can also do like TexturePacker, and add an "Auto-detect animations" option in spritesDockWidget, if this option is checked, the exporter will detect the sprite frames that start with the same name and end with a different number, and group them together in <array>, like here ---> https://pastebin.com/W5CViZ55. The problem with this method is that the user have less control, he can't set the <delay> for example.

What do you think will be better ?

**Comment by amakaseev** [comment]
I think better to combine two ways
- automatic or manual definition of animation
- editing settings of each animation

**Comment by amakaseev** [comment]
TexturePacker keeps only spritesheet, without animation, even with included 'Auto-detect animation'

**Comment by NathV** [comment]
Yes i was wrong, they use a "getAnimation" method with cocos2d-x : https://www.codeandweb.com/texturepacker/tutorials/animations-and-spritesheets-in-cocos2d-x

**Comment by amakaseev** [comment]
I have already begun to expand animation preview dialog for creation of animation
possibly it will be useful

**Comment by NathV** [comment]
I am working on it to 
<a href="http://i.imgur.com/IAsGr2c.png">
  <img src="http://imgur.com/IAsGr2cl.png" />
</a>

**Comment by amakaseev** [comment]
please look at the last changes
I already have almost made everything, it was necessary to make only save and publish

**Comment by amakaseev** [comment]
<img width="1257" alt="screenshot 2017-09-01 21 26 19" src="https://user-images.githubusercontent.com/9881001/29982966-6f4f7aa6-8f5c-11e7-8c7a-cc793cdef0bc.png">

**Comment by amakaseev** [comment]
if you have a desire to expand the editor of animation and to add publishing, would be remarkable
